### ISSUE TITLE
Event Channels may fail when source cluster information is lost or removed

### ISSUE THREAD

**Issue by brianoliver** [coherence-community/coherence-incubator#156]
Event Channels will fail to distributed Cache Entries when their Source Cluster Decorations are deliberately removed, accidentally corrupted (due to a custom binary value update) or lost (due to them being stripped from a binary entry during custom binary-based entry recovery).

For example, the following exception has seen to occur.
```
Class:com.oracle.coherence.patterns.eventdistribution.distributors.AbstractEventChannelController Method:onDistribute
java.lang.IllegalStateException: Can't extract the CLUSTER_INFO_DECORATION_KEY from the EntryEvent. The BinaryValue is not decorated
        at com.oracle.coherence.patterns.eventdistribution.channels.AbstractInterClusterEventChannel.getSourceClusterMetaInfo(AbstractInterClusterEventChannel.java:178)
        at com.oracle.coherence.patterns.eventdistribution.channels.AbstractInterClusterEventChannel.isDistributable(AbstractInterClusterEventChannel.java:203)
        at com.oracle.coherence.patterns.eventdistribution.channels.AbstractInterClusterEventChannel.getEventsToDistribute(AbstractInterClusterEventChannel.java:270)
        at com.oracle.coherence.patterns.eventdistribution.channels.RemoteClusterEventChannel.send(RemoteClusterEventChannel.java:292)
        at com.oracle.coherence.patterns.eventdistribution.distributors.AbstractEventChannelController.onDistribute(AbstractEventChannelController.java:1027)
        at com.oracle.coherence.patterns.eventdistribution.distributors.AbstractEventChannelController.access$200(AbstractEventChannelController.java:90)
        at com.oracle.coherence.patterns.eventdistribution.distributors.AbstractEventChannelController$5.run(AbstractEventChannelController.java:765)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
```

When such an exception occurs, the Event Channel will become paused as the causing cache entry will remain on the replication queue.   Instead the cache entry should be ignored (and logged), thus allowing replication to continue.

**Comment by brianoliver** [comment]
Resolved in 11.2.3-SNAPSHOT

**Comment by brianoliver** [comment]
Resolved in 11.3.3-SNAPSHOT

**Comment by brianoliver** [comment]
Resolved in 12.3.3-SNAPSHOT

**Comment by brianoliver** [comment]
Resolved in 13.0.0-SNAPSHOT
